source/nanoFramework.CoreLibrary/System/Guid.cs

```diff
@@ -118,6 +118,22 @@ public Guid(byte[] b)
             _k = b[15];
         }
 
+        /// <summary>
+        ///   Creates a new <see cref="Guid"/> based on the value in the string.  The value is made up
+        ///   of hex digits speared by the dash ("-"). The string may begin and end with
+        ///   brackets ("{", "}").
+        ///   
+        ///   The string must be of the form dddddddd-dddd-dddd-dddd-dddddddddddd. where
+        ///   d is a hex digit. (That is 8 hex digits, followed by 4, then 4, then 4,
+        ///   then 12) such as: "CA761232-ED42-11CE-BACD-00AA0057B223"
+        /// </summary>
+        /// <param name="g">String representation of new <see cref="Guid"/>.</param>
+        public Guid(string g)
+        {
+            if (!TryParseGuidWithDashes(g, out this))
```

Comment:
Following our coding style guidelines, open/close brackets must be there at all times. Even if it's a single statement.